config/config.go

```diff
@@ -137,7 +137,11 @@ func NewLoader(options ...LoaderOption) *Loader {
 		v: getViperWithDefaults(),
 		opts: []viper.DecoderConfigOption{
 			viper.DecodeHook(
-				StringToJsonFunc(),
+				mapstructure.ComposeDecodeHookFunc(
+					StringToJsonFunc(),
+					mapstructure.StringToTimeDurationHookFunc(),
+					mapstructure.StringToSliceHookFunc(","),
+				),
```

Comment:
Can we have test cases here so that this kind of bug can be found early? 🤔

Reply:
We can do that